Sources/CLI/Prompter.swift

```diff
@@ -18,12 +18,46 @@ private extension URL {
 }
 
 enum Prompter {
+	enum TicketService: Int, CaseIterable {
+		case jira
+		case githubIssues
+		case none
+
+		fileprivate var name: String {
+			switch self {
+			case .jira:
+				return "Jira"
+			case .githubIssues:
+				return "GitHub Issues"
+			case .none:
+				return "None"
+			}
+		}
+
+		fileprivate static var promptDescription: String {
```

Comment:
<!--
  1 failure:  `fileprivate` sho...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: Sources/CLI/Prompter.swift;
  Line: 37;
-->
- :no_entry_sign: `fileprivate` should be avoided. (`strict_fileprivate`)